src/ontology/mondo-edit.obo

```diff
@@ -448018,7 +448021,7 @@ xref: Orphanet:972 {source="MONDO:equivalentTo"}
 xref: UMLS:C1834559 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:331775"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
 is_a: MONDO:0005336 ! myopathy
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="OMIM:160120", source="Orphanet:972"} ! KCNA1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="Orphanet:972"} ! KCNA1
```

Comment:
this is still on the website

Reply:
this should have been maintained `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="OMIM:160120", source="Orphanet:972"} ! KCNA1` 
and is in the omim.owl and mondo-omim-genes.robot.tsv files we create